src/Workspaces/SharedUtilitiesAndExtensions/Workspace/Core/Extensions/SyntaxGeneratorExtensions.cs

```diff
@@ -58,10 +59,11 @@ private static SyntaxNode CreateNotImplementedException(SyntaxGenerator codeDefi
             ITypeSymbol type)
         {
             var equalityComparerType = compilation.EqualityComparerOfTType();
-            var constructedType = equalityComparerType.Construct(type);
-            return factory.MemberAccessExpression(
-                factory.TypeExpression(constructedType),
-                factory.IdentifierName(DefaultName));
+            var typeExpression = equalityComparerType == null
+                ? factory.GenericName(nameof(EqualityComparer<int>), type)
+                : factory.TypeExpression(equalityComparerType.Construct(type));
```

Comment:
product bug.  we weren't resilient to not finding `EqualityComparer<T>` in our cor library and would crash.